src/openrct2/object/ImageTable.cpp

```diff
@@ -443,15 +443,23 @@ std::vector<std::pair<std::string, Image>> ImageTable::GetImageSources(IReadObje
     return result;
 }
 
-void ImageTable::ReadJson(IReadObjectContext* context, json_t& root)
+bool ImageTable::ReadJson(IReadObjectContext* context, json_t& root)
```

Comment:
Instead of returning the state this way you could just set a member variable of the image table to say its using fallbackSprites.